### PR TITLE
Skilambi943 caption

### PR DESCRIPTION
**Pull request by skilambi943**

### CODE REVIEW

File: _slides/09-skilambi943.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+this is sailaja--how are you all??
```
Comment (vkanithi) [inline _slides/09-skilambi943.md:6]: 💯

Comment (vkanithi) [inline _slides/09-skilambi943.md:6]:
```suggestion
This is Sailaja -- how are you all?
```

**Review by vkanithi — CHANGES_REQUESTED** [review]

File: _slides/09-skilambi943.md (hunk 1 above)
Comment (ericalloyd) [reply _slides/09-skilambi943.md:6]: Hey, thanks for catching that! Will change

Comment (skilambi943) [reply _slides/09-skilambi943.md:6]: very good suggestion!! but i like 2 ??

Comment (vkanithi) [inline _slides/09-skilambi943.md:6]:
```suggestion
This is Sailaja--how are you all??
```

**Review by vkanithi — APPROVED** [review]

**Review by ericalloyd — APPROVED** [review]

**Pull request merged**